src/rebuild/README.md

```diff
@@ -1,275 +1,150 @@
-# Self-healing
+#Self-healing (Rebuild)
 
-**Topiccs in this Chapter:**
+In DAOS, if the data is replicated with multiple copies on different
```

Comment:
maybe can change it a little bit to make the description be generic for both replica and EC

Reply:
Not sure about EC yet, since we do not have it yet. Maybe add it once we implemented it.

Reply:
Oh, we still do not have it yet, I will add it once we finish that, since I am not sure EC rebuild will be under this directory.

Reply:
yes we should add it.